src/Entity/File.php

```diff
@@ -78,4 +78,47 @@ public function setDimensions(?array $dimensions): void
     {
         $this->dimensions = $dimensions;
     }
+
+    /**
+     * A simple shortcut to the image width.
+     * Similar to `$file->getDimensions()[0]`.
+     *
+     * @return int|null Returns `null` if dimensions array is itself null
+     */
+    public function getWidth(): ?int
+    {
+        return $this->getDimensions() ? $this->getDimensions()[0] : null;
+    }
```

Comment:
We can access directly the property. Moreover, we should avoi implicit casting to Boolean and check for null equality.

Reply:
You mean I need to do `null !== $this->getDimensions()`?

Reply:
No, I mean `null !== $this->dimensions`

Reply:
Oh, yes, you're right. I'm stupid 🤦‍♂️
As I based this on my own code, I didn't noticed I was still use the getter, sorry.
I'll fix that at once :)